exercises/more_functional/problem.md

```diff
@@ -0,0 +1,39 @@
+# Let's tweak/refactor the previous lesson to be more declarative!
```

Comment:
I'm planning on dropping this exercise. See #48. What do you think?

Plus this line should probably be "more functional!"